test.sh

```diff
@@ -56,6 +56,31 @@ function test_metrics() {
 
 # ==================================================================================
 
+function test_entrypoint() {
+  local name=$1
+  local entrypoint=$2
+
+  local container_id=$(docker run --name ${name}-test -d \
+                         -e LANG=en_US.UTF-8 -e PARAMETER_THAT_MAY_NEED_ESCAPING="&'\"|< é\\(" ${name} \
+                         ${entrypoint} --commandLineArgValueThatMayNeedEscaping="&'\"|< é\\(" --killDelay=1 --exitCode=0)
```

Comment:
not sure what `--killDelay=1 --exitCode=0` does / is needed for... `docker run` or `spring-cloud-deployer-spi-test-app-1.3.4.RELEASE-exec.jar` arguments? Perhaps worth an inline comment to clarify to future readers.

Reply:
--killDelay=1 --exitCode=0 are arguments for the test application to define if the the app should be killed after running its test and what exitcode should be returned. i'll add some comments